domains/sarthak.json

```diff
@@ -0,0 +1,12 @@
+{
+        "owner": {
+           "username": "dubey0613",
+           "email": "dubey.sarthak0613@gmail.com",
+           "discord": "877525618525364295"
+        },
+    
+        "record": {
+            "CNAME": "https://sarthak-portfolio-web.vercel.app/"
```

Comment:
`CNAME` can't contain `/` nor protocol (https://)
and we do not support vercel atm

Reply:
Kindly check once again!